content/reference-strategy/index.md

```diff
@@ -0,0 +1,142 @@
+---
+rfc:
+start_date: 2018-03-29
+pr:
+status: draft
+---
+
+# Reference strategy
+
+## Summary
+
+This RFC proposes CURIEs as the only way to reference between records and from a record
+to a register (set of records). It can be seen as the natural consequence of
+[RFC 0002: URI Consolidation](https://github.com/openregister/registers-rfcs/blob/master/content/uri-consolidation/index.md).
+
+Note that other types of references (e.g. subset of records) are out of the
+scope. See the discussion [Group References](https://github.com/openregister/registers-rfcs/issues/4)
+for more on this.
+
+## Motivation
+
+Currently, there are two mechanisms for referencing: a foreign key and a CURIE.
+
+### Foreign Key
+
+A foreign key is a regular string that _happens_ to be an identifier in
```

Comment:
Thinking about this, it connects with another discussion about expectations around primary key values: #18 

In there I was only considering strings but your comment opens the door to more (potential) complexity.

I think we care about how flexible we must be with identifiers that can be used in CURIEs (and because historical reasons in foreign keys). In particular with CURIEs, the value needs to be encoded in terms of a URL path so `foo:1` would be the same if the identifier is a string or an integer. I'm leaning towards exploring issues derived to restrict identifiers to be only a subset of UTF-8.